backend.ps1

```diff
@@ -14,14 +14,15 @@ class Instance {
         Start-Transcript -Path $transcriptPath -Force
```

Comment:
Before reading the following pin pointed things, make sure you pay more attention to:
1. Consistency
2. Consistency
3. Lisibility (no %, or formatting strings using the concat operator +)